public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -62,153 +77,161 @@ code-example(language="bash").
   We don't even have a web server that can handle requests for heroes.
   Until we do, *we'll have to fake it*.
 
-  We're going to *trick* the http client into fetching and saving data from
-  a demo/development service, the *in-memory web api*.
+  We're going to *trick* the HTTP client into fetching and saving data from
+  a mock service, the *in-memory web API*.
 
   The application itself doesn't need to know and shouldn't know about this.
-  So we'll slip the *in-memory web api* into the configuration *above* the `AppComponent`.
+  So we'll slip the in-memory web API into the configuration *above* the `AppComponent`.
 
   Here is a version of `main` that performs this trick
-+makeExample('toh-6/ts/app/main.ts', 'final', 'app/main.ts (final)')(format=".")
-
-:marked
-  We're replacing the default `XHRBackend`, the service that talks to the remote server,
-  with the *in-memory web api* service after priming it with the following `in-memory-data.service.ts` file:
-+makeExample('toh-6/ts/app/in-memory-data.service.ts', null, 'app/in-memory-data.service.ts')(format=".")
-:marked
-  This file replaces the `mock-heroes.ts` which is now safe to delete.
++makeExcerpt('app/main.ts', 'final')
 
-.alert.is-helpful
+block backend
   :marked
-    This chapter is an introduction to the Angular http client.
-    Please don't be distracted by the details of this backend substitution. Just follow along with the example.
+    We're replacing the default `XHRBackend`, the service that talks to the remote server,
+    with the in-memory web API service after priming it as follows:
 
-    Learn more later about the *in-memory web api* in the [Http chapter](../guide/server-communication.html#!#in-mem-web-api).
-    Remember, the *in-memory web api* is only useful in the early stages of development and for demonstrations such as this Tour of Heroes.
-    Skip it when you have a real web api server.
++makeExample('app/in-memory-data.service.ts', 'init')
 
+p This file replaces the #[code #[+adjExPath('mock-heroes.ts')]] which is now safe to delete.
+
+block dont-be-distracted-by-backend-subst
+  .alert.is-helpful
+    :marked
+      This chapter is an introduction to the !{_Angular_http_library}.
+      Please don't be distracted by the details of this backend substitution. Just follow along with the example.
```

Comment:
I agree, but that was in the original TS text so I left it as is (but excluded it from the Dart edition). Remember that for this time, my goal was mainly to proof read, not a general edit.